### ISSUE TITLE
[Tropical Biome]Animals to add

### ISSUE THREAD

**Issue by kaptain-kavern** [kaptain-kavern/EcosystemOverhaul#1]
- [x] ~~Tiger~~
- [x] ~~White Tiger~~
- [x] ~~Panda Bear~~
- [x] ~~Jaguar~~
- [x] ~~Tapir~~  
- [x] ~~Peacock~~
- [x] Komodo Dragon [Graphic Placeholder]
- [x] Python [Graphic Placeholder]  
- [x] Squirrel Monkey [Graphic Placeholder]
- [x] Toucan [Graphic Placeholder]  
- [x] Gorilla [Graphic Placeholder]  
- [x] Spider Monkey [Graphic Placeholder]  
- [x] Capuchin Monkey [Graphic Placeholder]  
- [ ] Orangutan
- [ ] Anteater
- [ ] Sloth
- [ ] Macaw  
- [ ] Parrot  
- [ ] Ocelot  
  ~~Need to add the others new animals planned~~

**Comment by kaptain-kavern** [comment]
_Update :_ Toucans, Squirrel Monkeys and Peafowls are in (with graphic Placeholders)

**Comment by lpCoercion** [comment]
I added the wild boar, rat and tortoise back into the biome.

**Comment by kaptain-kavern** [comment]
Wow! With the damage increased those tigers are really dangerous lol. Second time on of my colonists is one shot downed by a a Tiger ^^

**Comment by lpCoercion** [comment]
that is strong lol

do you think it is too strong or do you like it that way? I personally like the added danger and think it makes the game choices more meaningful, but we can definitly lower the damage if it is too much. so you know, the damage is almost doubled if they get a suprise attack on you.

**Comment by kaptain-kavern** [comment]
Once you know it you plan more carefully :warning: 

No it's ok, we just need to pay attention to not overpowered them regarding to vanilla values (like having our Jaguar hitting harder than a vanilla elephant :laughing: )

---

I've just added the Tapir (https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/26fe5f55062f1310538b77d58d05a71be3a660bb) but it still need the "facing" texture :wink:

**Comment by lpCoercion** [comment]
Should we modify the vanilla animals as well? make the elephant have a lot more health/ armor for example?

**Comment by lpCoercion** [comment]
Use these for the tapir textures. They are updated.

![tapir_side](https://cloud.githubusercontent.com/assets/22029252/18838639/289ef236-83d6-11e6-8353-eadcc280faaf.png)

![tapir_front](https://cloud.githubusercontent.com/assets/22029252/18838794/ab8427d4-83d6-11e6-98c6-f3402e14f515.png)

![tapir_back](https://cloud.githubusercontent.com/assets/22029252/18838876/f4bf23b8-83d6-11e6-989b-9d15f7842b7d.png)

**Comment by kaptain-kavern** [comment]
They are in - https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/f2a0485d0fb80a498cd378ce1ff99949f2ede871

**Comment by lpCoercion** [comment]
Junglefowl

![junglefowl_side](https://cloud.githubusercontent.com/assets/22029252/18842894/a34d3566-83e4-11e6-933b-61bdeaa45549.png)

![junglefowl_front](https://cloud.githubusercontent.com/assets/22029252/18842908/a5f1829a-83e4-11e6-9c2f-332b3dbf7de2.png)

![junglefowl_back](https://cloud.githubusercontent.com/assets/22029252/18842910/a73b6c38-83e4-11e6-801d-167e97cb0630.png)

Rarity: Common
MarketValue: 50
MoveSpeed: 3.5
Wild: 45
Obedience: N
Release: N
Rescue: N
Haul: N
ManhunterFailTame: 0%

**Comment by kaptain-kavern** [comment]
I'm typing the code atm (mixing values you provide with Turkey/chicken code). In core with Turkey and chicken, there is also a separate texture for adult females (hen), is it something you've planned or have thing about?
![turkeyhen_side](https://cloud.githubusercontent.com/assets/2794612/18843488/8a72d6b0-8419-11e6-819d-b61b21898cc8.png) / ![turkeytom_side](https://cloud.githubusercontent.com/assets/2794612/18843492/9092ac5a-8419-11e6-8c56-836535779462.png)

**Comment by lpCoercion** [comment]
I thought about it, but I cant find the hen or chick texture. I will see what I can find.

If not, I will just edit these textures for hens.

**Comment by kaptain-kavern** [comment]
I may have been wrong about the chicken/hen, I only found textures for Turkey/hen but I remembered noticing that in the code (it was with cow/bull).  
We even could have separate texture for lifestages also (even different M/F for each Lifestage). I wasn't sure you were aware of that possibility with the XML code

**Comment by lpCoercion** [comment]
That is pretty cool. do you think that we should focus on adding different types of animals first or should we add gender/lifestage textures as animals are added?

**Comment by kaptain-kavern** [comment]
IMHO it's just "minor" improvements, it can be more useful, regarding balancing, to focus on missing animals and plants. That way we can reach a "mature" point sooner, and then, can focus on improvements and final balancing.

But hey it really depends on how  you feel it. I mean, me I can focus on any part of the code at any moment; I really don't know how it's work in your mind for the drawing, I could totally understand focusing on other things just because you had inspiration for drawing this at that particular moment.

**Comment by kaptain-kavern** [comment]
Junglefowl and Tapir are  :ok:.

On [dev](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/tree/dev)

**Comment by lpCoercion** [comment]
alright cool. Should we do a v1.2 just adding them to increase prey variety?

**Comment by kaptain-kavern** [comment]
This time it could even be the 1.1.1 :wink: because this one will be savecompatible. The game will introduce the new animals the next time he check for filling the biome

**Comment by lpCoercion** [comment]
alright lets do that then :)

let me know when its up

**Comment by lpCoercion** [comment]
Also, if you can, add the peafowl to 1.1.1 :)

![peafowl_side](https://cloud.githubusercontent.com/assets/22029252/18847764/69fcec88-83f9-11e6-8923-8da8e88e53ec.png)

![peafowl_front](https://cloud.githubusercontent.com/assets/22029252/18847766/6d34dd20-83f9-11e6-9296-d603076c0639.png)

![peafowl_back](https://cloud.githubusercontent.com/assets/22029252/18847769/6f995208-83f9-11e6-9fee-1b1f738172f1.png)

**Comment by kaptain-kavern** [comment]
Seriously this one is beautiful man (I mean all are but I very like this one). 

I'm still struggling a bit with the Junglefowl file dimension. I had to resize them because it was weird (the size were not appearing equal if the fowl was facing or siding).  Turns out that apparently all 3 files have to have the same dimension 64x64 or 128x128 (I'm not sure if any values are possible), but the same for all three.

So yes I will release after including the peafowl :bird:

**Comment by lpCoercion** [comment]
thanks :)

The peafowl should have the same stats as the junglefowl, but be very rare and have a marketvalue of 2800

let me know if you need me to fix the sizes. I can add them all to the same size blank background.

**Comment by lpCoercion** [comment]
If you havent released yet, add the Okapi too. I think this should be v1.2 since we are adding so much content.

![okapi_side](https://cloud.githubusercontent.com/assets/22029252/18850941/bdd5dc76-8407-11e6-9699-a04633b2d591.png)

![okapi_front](https://cloud.githubusercontent.com/assets/22029252/18850946/c0f98290-8407-11e6-8077-3b74b5bb0ed1.png)

![okapi_back](https://cloud.githubusercontent.com/assets/22029252/18850947/c360b63e-8407-11e6-97fd-2f4f2585e5c2.png)

**Comment by kaptain-kavern** [comment]
Wow you're on fire today :fire_engine: 
I like very much the okapi too

---

![Imgur](http://i.imgur.com/owttKhX.png?1)

How about that ?

**Comment by kaptain-kavern** [comment]
![Imgur](http://i.imgur.com/GW6S3rH.png?1)

**Comment by lpCoercion** [comment]
looks good to me :)

let me know when you get v1.2 released. I will have the bongo next.

**Comment by lpCoercion** [comment]
The tapir may be a little big though. what do you think?

**Comment by kaptain-kavern** [comment]
I think the same. I'll reduce it's size a bit and do the release before coming back here to let you know

**Comment by kaptain-kavern** [comment]
While i'm in the code : Should our mammals (currently Okapi and Tapir - _I may forget some_) be milkable ?

---

#### Resized Tapir

![Imgur](http://i.imgur.com/VzhXcV2.png?1)

**Comment by kaptain-kavern** [comment]
![Imgur](http://i.imgur.com/VzhXcV2.png?1)

**Comment by lpCoercion** [comment]
Yeah. Great idea.

size looks good :)

**Comment by kaptain-kavern** [comment]
Ok i'll do that and we finally release ;op then

**Comment by lpCoercion** [comment]
by the way. okapi and tapir: both common

**Comment by kaptain-kavern** [comment]
Like that : 

```
         <!-- Common -->
         <Chinchilla>2.5</Chinchilla>
         <Capybara>2.2</Capybara>
         <Rat>1.8</Rat>
         <WildBoar>1.5</WildBoar>
         <Tortoise>1.4</Tortoise>
         <Tapir>1</Tapir>
         <Okapi>0.8</Okapi>
         <Junglefowl>1.5</Junglefowl>
         <!-- <Anteater>1</Anteater> -->
         <!-- <Sloth>0.8</Sloth> -->
         <!-- <CapuchinMonkey>1.6</CapuchinMonkey> -->
         <!-- <SpiderMonkey>1.6</SpiderMonkey> -->         
         <!-- <SquirrelMonkey>1.6</SquirrelMonkey> -->
```

seems ok?

**Comment by lpCoercion** [comment]
What do you think of this?

Okapi 1.8 
Tapir 1.5 
Tortoise 1 
Rat 1 
! Anteater 1.8

**Comment by kaptain-kavern** [comment]
Oki doke.

I just do the test for knowing if it save compatible or not (i'm not that sure now that I changed Tapir to been milkable) and do the release in a couple of minutes

**Comment by lpCoercion** [comment]
by the way: okapi 75 wild

**Comment by kaptain-kavern** [comment]
Had a problem/bug : I previously had set the code for smallbirdbase to not dropping leather but this caused today's bug. Now resolved (they now dropped very little amount of leather). I'm doing the release now

**Comment by kaptain-kavern** [comment]
Released as 1.2 : --------------> https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/releases/latest

I mean it is save compatible if you don't mind the 10/15 red error messages and if you destroy some items from stockpiles. Too complicated to explain in case of complain ... So :arrow_right: **1.2** :smile:

**Comment by lpCoercion** [comment]
awesome :)

**Comment by kaptain-kavern** [comment]
I'm on the Bongo code ^^

**Comment by lpCoercion** [comment]
This is my first attempt at the animal list. Let me know what you think.

[Animal Stats.xlsx](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/files/498174/Animal.Stats.xlsx)

**Comment by lpCoercion** [comment]
another option is to make the dmg/health match vanilla values

**Comment by kaptain-kavern** [comment]
I'm ok with the stats I saw ;)  Like I stated previously i'm not against "justified" heavier damage (as long as it still on par with vanilla "logic" : no birds should hit more harder than an elephant per example). What can also be done is having heavy damage but longer cool down (I noticed that in the code for elephants).

For coding the Bongo I had to make an entirely custom BodyDef (for it to spawn 2 horns when killed); it takes me some time but I think I understand more how it works now  - just so you know that we can now have that option too : (dropping bodyparts) in addition to shearing and milking.

Speaking of that : how about shearable chinchillas ? ^^ (more for colder biomes maybe ^^ )

**Comment by lpCoercion** [comment]
We can make shearabke chinchillas but they wont provide much fur lol

**Comment by lpCoercion** [comment]
I think people get confused when you post screenshots of placeholder art.

**Comment by kaptain-kavern** [comment]
Idk what I was thinking when I propose that. Chinchillas are bred for their furs (leather + hair), so there is no point to make them shearable in game don't you think?

If i'm about to modify vanilla animals I prefer to do it at once, so here is a list for us to discuss : 

| Animal | Make "milkable" | Make "shearable" |
| :-: | :-: | :-: |
| Deer | Yes 4milk/2days | No |
| Ibex | Yes 4milk/2days | _Could drop a little_ |
| Alpaca | Yes 4milk/2days | Already |

Some other (minor)  ideas I got while browsing wikipedia : 
- Chinchillas should be herd animals
- Gazelles and Ibexes could also drop horns; **and** Elks and Caribous  could drop anters (if you're ok with the way I've done bongos) - _How I see things : In the base mod, dropped horns and anters are only meant to be sold and we could have recipes using them in the "hunter mod/pack"_
- Pigs also produce milk but it is hard to obtain :wink:  (could produce a little every X days to reflect that)

Another thing I think about is try to "normalize" leather and meat for them to be more easily managed in stokpiles.  Like all birds (or most) dropping bird meat and bird leather against a leather type and a meat type for each and every animals we will add.  I'm not sure it's clearly explained lol sorry

**Comment by lpCoercion** [comment]
Good ideas.

I would like to keep unique names for leathers, meats, items, etc. (like in vanilla), because when you make things it uses the the title (e.g. tiger leather sandbag, chinchilla fur coat, cow skin pants, etc.)

Some other ideas:
- all birds drop feathers
- tusks from wild boar
- venom from cobra (anti-venom recipe "CK Drugs")
- all insectoids drop chitin

**Comment by kaptain-kavern** [comment]
Yep I understand that :wink: keep the idea in mind though because if we introduce a ton of say birds (any variation on little animals) we will end up with stockpile full of very little amount of different-but-similar leather types or meat. It was more an idea for the future maybe, lol. And also not something we have to do for every animals. I was specifically thinking of birds (and not all) and monkeys (not all either) per example.

---

I'm all for adding more dropping items no problem. What I think we should keep in mind is to clearly think which we will add to the base mod (I was thinking any items that can be directly used in game) and those we will released as a more elaborate, distinct mod/pack/add-on.

Like feathers by themselves, I don't see them being usable as is (not to mention how to integrate them with the way bodydef and bodypartdef works ^^) But tusks can be item "just-to-be-sold" and have more purpose (crafting furnitures/decorations maybe) in the "Hunter's mod project".  
If I can code anti-venom (I think something that boost "blood filtration" but had to do more research) I will be super happy, because I like the idea !

What do you think?

_edit:_ I just think about [fermented milk](https://en.wikipedia.org/wiki/Kefir) also (could maybe even boost health a bit) - _Easy recipe - quickly and easily "_**add-able**_"_

**Comment by lpCoercion** [comment]
I think the recipe for fermented milk would be hard as we have no yeast, grain, etc.

what would blood filtration do in game? what do you think anti venom should do in terms of gameplay? would we need cobra posion and anti venom?

**Comment by kaptain-kavern** [comment]
Oh I was thinking about keep things simple and make it from milk only (in the game you make beer without the need to add anything - speaking about that I checked and the all wort fermenting in barrel system is all hard coded in C# so I can't use and tweak that).

Blood filtration increase toxin/illness elimination. There already is venom from serpent bites in the game. All I have to do is make a consumable that have an (negative) effect on the _toxic_ _hediff_ (I know that part of the code better because of my previous research when I've done DESurgeries and A Dog Said mods)

Edit : another ideas again. Have coconuts harvested as raw but having to open them before consuming (just a simple recipe at stoves or crafting spot or both, using raw coconuts to spawn milk and flesh)

**Comment by lpCoercion** [comment]
yeah fermented milk could work. same with fermented fruit (wine/hooch)

oh ok. that works for the venom.

I would like to keep things simple, so I think processing items (like coconuts or pelts) is too much.

**Comment by kaptain-kavern** [comment]
With the next released in mind (new plants, crops, recipes, etc),  I went nuts and [progressed on several "fronts" yesterday](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commits/dev), here is an update because I want to know if it suits you : 
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/545c3aa3afa73bde0d02a4425b3bd7beb8af4990 - Bongo's code is in (still need proper textures - use gazelle one for now if you test)
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/0e4271d3aae0701be157e14dc581e2446750d4a3 - [Optional Commit] Eating-trees-elephants for test (if you can test it and see what you think - else it's meant to be easily reversed/erased by just having the elephant code erased in the Vanilla_Override.xml file)
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/cbfe3d9dbcb86caec66b0cdef2462fcd33868b7f - Overriding vanilla rhino for adding a dropable horn
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/bdb1f00482877f86112852d738ba359a850d4640 - Modded vanilla Ibex, deer, alpaca and chinchilla : Ibex, deer, alpaca are now producing milk / Chinchillas are now herd animals
- https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/1f3dd00cfaff3dd6e949a6c73d7fb4096765adae - Junglefowl now lay unfertilized egg like chicken in Vanilla... Meanings they're more suitable for being used as farm animals but they're far from lay as often as chicken and also means that eggs can now be foraged too (they should lay in the wild too but I haven't test it long enough yet)

**Comment by lpCoercion** [comment]
looks good :)

Can you please check out the checklist issue?

https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/issues/2#issuecomment-250930615

**Comment by lpCoercion** [comment]
Rhino Horn: ![rhinohorn](https://cloud.githubusercontent.com/assets/22029252/19018310/b83f78c8-8829-11e6-99ff-8a97e4a4c4fa.png)
Gazelle Horn: ![gazellehorn](https://cloud.githubusercontent.com/assets/22029252/19018327/96dba28c-882a-11e6-9364-506fe99d45bd.png)
Ibex Horn: ![ibexhorn](https://cloud.githubusercontent.com/assets/22029252/19018328/9a3a822c-882a-11e6-96ec-159373e18a06.png)
Caribou Antler: ![caribouhorn](https://cloud.githubusercontent.com/assets/22029252/19018381/b0d60dba-882c-11e6-8d1e-1e146eac0f49.png)
Deer Antler: ![deerhorn](https://cloud.githubusercontent.com/assets/22029252/19018382/b5e919be-882c-11e6-90b7-2691fb264685.png)
Elk Antler: ![elkhorn](https://cloud.githubusercontent.com/assets/22029252/19018384/ba97203c-882c-11e6-87b6-3bf60528b9c1.png)

**Comment by kaptain-kavern** [comment]
Ok they're all in now :wink:  with the new modded bodydefs (one for animal with horns and another for animal with antlers) and the code for the items are in _Items_Exotic.xml_ (https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commits/dev)

![Imgur](http://i.imgur.com/RBrnyLS.png?1)

| Animal horn/antler | Damage | Cooldown | Price |
| :-: | :-: | :-: | :-: |
| Rhino | 15 dmg | 2.4sec | 450 silver |
| Gazelle | 10 dmg | 1.8sec | 200 silver |
| Bongo | 12 dmg | 2sec | 250 silver |
| Ibex | 14 dmg | 2.2sec | 100 silver |
| Deer | 10 dmg | 2sec | 100 silver |
| Elk | 12 dmg | 2.2sec | 150 silver |
| Caribou | 15 dmg | 2.4sec | 250 silver |

**Comment by lpCoercion** [comment]
awesome :)

**Comment by lpCoercion** [comment]
have eggs been added to the peafowl and junglefowl?

also, in game I noticed that a peahen had a junglefowl texture

**Comment by kaptain-kavern** [comment]
Yes, eggs definitions are located [in Items_Resource_Eggs.xml](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/blob/dev/Defs/ThingDefs_Items/Items_Resource_Eggs.xml#L112).

For the junglefowl/peahen it's weird, I double checked and all seems well defined. But I thought maybe I named wrongly some of the lifestages and/or gender. 

Here is what I've done (tell me if anything is wrong or to be changed)

| `<Defname>` | `<Label>` | `<labelMale>` | `<labelFemale>` | `young animal <label>` |
| :-: | :-: | :-: | :-: | :-: |
| Peafowl | peafowl | peacock | peahen | peachick |
| Junglefowl | junglefowl | jungle peacock | jungle peahen | jungle peachick |

Ho and also, the young ones are using the vanilla texture for chicks too. I tried to reflect the fact that IRL all chicks look the same and it's as they grow that their distinctive color take place.   I can reverse that and make it that for all lifestage it's the adult texture used (maybe it was that you saw?)

**Comment by lpCoercion** [comment]
ah yea. I think I saw jungle "peahen". 

maybe to avoid confusions call the junglefowl: junglefowl (male)/ jungle hen (female)

**Comment by kaptain-kavern** [comment]
~~Will do~~ **[Done](https://github.com/kaptain-kavern/CK_AnimalPlant_Pack/commit/7874db4342f9bdd8c87dd53b450e6c3668df13d1)** :+1: 

---

In the mean time ... how about that !?!?!?
![Imgur](http://i.imgur.com/rezK8gE.png?1)

**A Dog Said** compatibility patch

**Comment by lpCoercion** [comment]
that is awesome :D

**Comment by kaptain-kavern** [comment]
_**quick note** :_ I don't know if you have play with the on-going version, but Chinchilla seems too numerous. Maybe 2.5 is a bit high

**Comment by lpCoercion** [comment]
yeah. 

I think that the okapis may be a bit to common too?

**Comment by lpCoercion** [comment]
I will probably make a new texture for the cobra next release. is it possible to add a cobra venom too?

**Comment by kaptain-kavern** [comment]
Yep. Can do. No problems. Vanilla snakes already have poisoning bites so I can easily use or duplicate this effect

**Comment by lpCoercion** [comment]
I think pandas are way too common (should go from .3 to like .02)

**Comment by lpCoercion** [comment]
Jungle Hens

![junglehen_side](https://cloud.githubusercontent.com/assets/22029252/20055762/c317d3e6-a4b0-11e6-871e-e211bb7bda64.png)

![junglehen_back](https://cloud.githubusercontent.com/assets/22029252/20055763/c31cb262-a4b0-11e6-97a5-385e9b681273.png)

![junglehen_front](https://cloud.githubusercontent.com/assets/22029252/20055764/c31f1a66-a4b0-11e6-8a39-cc1989a8803a.png)

**Comment by lpCoercion** [comment]
Junglehen

![junglehen_back](https://cloud.githubusercontent.com/assets/22029252/20056004/026c17ae-a4b2-11e6-9301-8f965cc51e67.png)

![junglehen_front](https://cloud.githubusercontent.com/assets/22029252/20056006/02a5f460-a4b2-11e6-985f-b0bef5fa39e7.png)

![junglehen_side](https://cloud.githubusercontent.com/assets/22029252/20056005/02747fa2-a4b2-11e6-8ffe-c7eec0980ce9.png)

Cobra retexture

![cobra_back](https://cloud.githubusercontent.com/assets/22029252/20068205/df6976aa-a4e5-11e6-917f-b43b41ef9460.png)

![cobra_front](https://cloud.githubusercontent.com/assets/22029252/20068204/df67c062-a4e5-11e6-9881-a470efdc4fd4.png)

![cobra_side](https://cloud.githubusercontent.com/assets/22029252/20068206/df6a63b2-a4e5-11e6-8ce1-75a27f472da6.png)

**Comment by kaptain-kavern** [comment]
OK also added. For the cobra, all I have to do is "change" Vanilla textures by this one? We don't change any _defs_, right?

I very much like it BTW ;-)

---

![Imgur](http://i.imgur.com/GCQZQqA.png?1) | ![Imgur](http://i.imgur.com/IeeAwnh.png?1)

**Comment by lpCoercion** [comment]
looks good :)

Yeah. we are just replacing the vanilla texture.

**Comment by lpCoercion** [comment]
retexture of the monkey:

![monkey_front](https://cloud.githubusercontent.com/assets/22029252/20213174/3d43970a-a7d5-11e6-866b-0fe12e8bfe54.png)

![monkey_side](https://cloud.githubusercontent.com/assets/22029252/20213175/3d443dcc-a7d5-11e6-9c18-2cefb32cb2ab.png)

![monkey_back](https://cloud.githubusercontent.com/assets/22029252/20213176/3d471510-a7d5-11e6-86d1-87d9251d58af.png)

And, yea. I know that the perspective is off on these, but it is how it was done in the vanilla texture and I dont really want to spend a bunch of time re-angling theses lol

I think the monkey should be very common and live in groups

**Comment by kaptain-kavern** [comment]
![Imgur](http://i.imgur.com/Z2ZFccg.png?1)

Just retexturing. Should we edit their defs too ?

Here are the Vanilla value

``` xml
  <ThingDef ParentName="AnimalThingBase">
    <defName>Monkey</defName>
    <label>monkey</label>
    <statBases>
      <MoveSpeed>4.84</MoveSpeed>
      <ComfyTemperatureMin>-5</ComfyTemperatureMin>
      <MarketValue>300</MarketValue>
    </statBases>
    <race>
      <body>Monkey</body>
      <petness>0.25</petness>
      <baseBodySize>0.35</baseBodySize>
      <baseHungerRate>0.25</baseHungerRate>
      <baseHealthScale>0.45</baseHealthScale>
      <foodType>OmnivoreRoughAnimal</foodType>
      <leatherLabel>monkeyhide</leatherLabel>
      <leatherColor>(124,53,29)</leatherColor>
      <leatherInsulation>0.75</leatherInsulation>
      <trainableIntelligence>Advanced</trainableIntelligence>
      <wildness>0.60</wildness>
      <nuzzleMtbHours>72</nuzzleMtbHours>
      <gestationPeriodDays>16</gestationPeriodDays>
      <lifeExpectancy>30</lifeExpectancy>
    </race>
  </ThingDef>

  <PawnKindDef ParentName="AnimalKindBase">
    <defName>Monkey</defName>
    <label>monkey</label>
    <race>Monkey</race>
    <combatPower>35</combatPower>
    <wildSpawn_spawnWild>true</wildSpawn_spawnWild>
    <wildSpawn_EcoSystemWeight>0.9</wildSpawn_EcoSystemWeight>
    <wildSpawn_GroupSizeRange>
      <min>3</min>
      <max>7</max>
    </wildSpawn_GroupSizeRange>
  </PawnKindDef>
```

Maybe add the "herd animal" tag?

**Comment by lpCoercion** [comment]
looks good :) yeah lets add herd value

**Comment by kaptain-kavern** [comment]
Done :+1: 
c0e3a3798d829bcede9e7ecc73d43ae64ff699e1
